arm_controller:
    type: position_controllers/JointTrajectoryController
    joints:
       - arm_1_joint
       - arm_2_joint
       - arm_3_joint
       - arm_4_joint
       - arm_5_joint
       - arm_6_joint
    constraints:
      goal_time: 0.6
      stopped_velocity_tolerance: 0.05
      hip: {trajectory: 0.1, goal: 0.1}
      shoulder: {trajectory: 0.1, goal: 0.1}
      elbow: {trajectory: 0.1, goal: 0.1}
      wrist: {trajectory: 0.1, goal: 0.1}
    stop_trajectory_duration: 0.5
    state_publish_rate:  50
    action_monitor_rate: 10
/gazebo_ros_control:
    pid_gains:
      arm_1_joint: {p: 5000.0, i: 0, d: 10.0}
      arm_2_joint: {p: 2000.0, i: 0, d: 10.0}
      arm_3_joint: {p: 2000.0, i: 0, d: 10.0}
      arm_4_joint: {p: 2000.0, i: 0, d: 10.0}
      arm_5_joint: {p: 2000.0, i: 0, d: 10.0}
      arm_6_joint: {p: 100.0, i: 0, d: 0}